### PR TITLE
[i18n] Updates in Russian .po (WX)

### PR DESCRIPTION
**Pull request by pm-kan**

### CODE REVIEW

File: Misc/I18N/ru/omegat_wx/omegat/project_save.tmx [hunk 1]
```diff
@@ -484,6 +484,14 @@ Unknown error</seg>
         <seg>&amp;Сравнение</seg>
       </tuv>
     </tu>
+    <tu>
+      <tuv lang="EN-US">
+        <seg>&amp;Compare...</seg>
+      </tuv>
+      <tuv lang="RU-RU" changeid="pm_kan" changedate="20190503T182658Z" creationid="pm_kan" creationdate="20190503T164526Z">
+        <seg>&amp;Сравнить...</seg>
```
Comment (dskrvk) [inline Misc/I18N/ru/omegat_wx/omegat/project_save.tmx]: Should line 484 become "Сравнить" as well?

File: Misc/I18N/ru/omegat_wx/omegat/project_save.tmx [hunk 2]
```diff
@@ -876,6 +892,14 @@ Unknown error</seg>
         <seg>&amp;Недавно использованные элементы</seg>
       </tuv>
     </tu>
+    <tu>
+      <tuv lang="EN-US">
+        <seg>&amp;Recent Safes...</seg>
+      </tuv>
+      <tuv lang="RU-RU" changeid="pm_kan" changedate="20190503T164422Z" creationid="pm_kan" creationdate="20190503T164422Z">
+        <seg>&amp;Недавно использованные контейнеры...</seg>
```
Comment (dskrvk) [inline Misc/I18N/ru/omegat_wx/omegat/project_save.tmx]: "Недавние контейнеры" for brevity?

File: Misc/I18N/ru/omegat_wx/omegat/project_save.tmx (hunk 1 above)
Comment (pm-kan) [reply Misc/I18N/ru/omegat_wx/omegat/project_save.tmx]: No, it shouldn't: `Compare...`  used in the `File` menu and describe "action", but `Compare` used in `View->Reports` submenu and describe report type. (Maybe translation for `File->Merge...` should be changed, but "Слить..." sounds awkwardly, while "Объединить..." is a bit different.)

File: Misc/I18N/ru/omegat_wx/omegat/project_save.tmx (hunk 2 above)
Comment (pm-kan) [reply Misc/I18N/ru/omegat_wx/omegat/project_save.tmx]: Good idea! Will change in the next update (for this change MFC variant and screenshots should be updated too).

**Pull request merged**